src/Blocks/custom/carousel/carousel-style.scss

```diff
@@ -0,0 +1,64 @@
+@import '~swiper/swiper.scss';
+
+.block-carousel {
+	&__item {
+		flex-shrink: 0;
+		width: 100%;
+		transition: 0.3s transform ease-in-out, 0.3s opacity ease-in-out;
+		display: flex;
+
+		&:not(.swiper-slide-active) {
+			opacity: 0.3;
+		}
+	}
+
+	&__item-inner {
+		align-items: center;
+		display: flex;
+		width: 100%;
+		justify-content: center;
+	}
+
+	.block-image__image {
```

Comment:
you can use https://infinum.github.io/eightshift-docs/docs/basics/blocks-wrapper#wrapperparentclass feature to wrap every item in it selector and remove referencing items directly here

Reply:
Again you resolved the comment without fixing it

Reply:
Turns out I didn't need that style in the end 😅

Regarding the `wrapperParentClass`, Grga suggested that I add a filter which would add the class to the Carousel's image children.

After we logged the output of `render_block_data` it turned out that the correct class was already being applied.

I need the selector because when I wrote it I didn't know about the bug with wrappers in Safari (where `flex-direction: column` was required for things not to be weird and distorted) and that one seemed to fix it.

Because the images were rendered as components, the inner structure of carousel items was correct, but I needed to target the images themselves (as a part of the image component), and I couldn't reach it "intuitively" with selectors like all the others in that file. Is there a way to do that or?